Sources/ApolloCodegenLib/Glob.swift

```diff
@@ -185,7 +186,15 @@ public struct Glob {
     }
 
     return OrderedSet<String>(directories.compactMap({ directory in
-      var path = directory.appendingPathComponent(lastPart).standardizedFileURL.path
+      // build directory URL up the the '*' in the pattern, and check if it is a symlink, if so skip it
+      let symCheckURL = directory.appendingPathComponent(lastDirComponent).standardizedFileURL
+      if let resourceValues = try? symCheckURL.resourceValues(forKeys: [.isSymbolicLinkKey]),
+         let isSymLink = resourceValues.isSymbolicLink,
+         isSymLink {
+        return nil
+      }
+      
+      var path = directory.resolvingSymlinksInPath().appendingPathComponent(lastPart).standardizedFileURL.path
```

Comment:
@AnthonyMDev Adding `.resolvingSymlinksInPath()` here is still necessary to resolve test use cases since the above code will never show the `/private/var/....` path as a symlink, but this call still removes the `/private` part of the path based on the [documentation](https://developer.apple.com/documentation/foundation/nsurl/1415965-resolvingsymlinksinpath) for the function call